ui/packages/catalog-ui-search/src/main/webapp/react-component/location/bounding-box.js

```diff
@@ -142,6 +135,17 @@ const BoundingBoxLatLonDms = props => {
       if (props.drawing) {
         setDmsError(initialErrorStateWithDefault)
       }
+      if (!dmsError.error) {
+        setDmsError(
+          validateGeo('bbox', {
+            dms: true,
```

Comment:
✏️  doesnt really matter but we could change this to `isDms` or something, so it's more obvious it's a boolean